src/tools/qt-gui/qml/KeyWindow.qml

```diff
@@ -116,13 +116,13 @@ BasicWindow {
 					metaValueField.placeholderText: valuePlaceHolder
 
 					metaNameField.onTextChanged:  {
-						if(metaName !== metaNameField.text){
+                        if(metaName.trim() !== metaNameField.text.trim()){
```

Comment:
You are right that most users spaces enter spaces unintentionally. (Especially the ones which will be removed with trim)

But they are valid parts of key names and as such there should be *some way* to enter them. Silently trimming them seems a bit dangerous. Maybe a popup? Or highlight them to reduce the chance that they are unintentional?